### ISSUE TITLE
Case study revision: dushizima

### ISSUE THREAD

**Issue by danielturek** [BIDS/repro-case-studies#54]
Your case study represents a useful and interesting example of reproducibility in applied scientific research, and we value this contribution.  Overall it’s excellent, and we’re requesting the addition of a Title, and some minor revisions to your Narrative and Diagram.

Introduction
- [x] Please provide a one sentence “title” for your case study (e.g., “Developing, testing, and deploying efficient MCMC algorithms for hierarchical models using R”, or an equivalent level of detail), at the very beginning of your case study, following the example below.

Narrative
- [x] Please read carefully checking for typos or grammatical errors.
- [x]  You only briefly mention Overleaf in the key tools section.  If this is truly a key tool, then please add the usage of it to your Narrative (possibly also in the relevant spot of your Diagram), and explain it’s usage and importance in some detail.

Diagram
- [ ] Please remove the little semi-circles in the lines, where they overlap.  Not necessary, it’s understandable where the lines are going.
- [x] Please make it so the lines from the top-left box (Sources) don’t cross.
- [x] Please adjust the lines surrounding the central box (Software Tools) so they don’t go behind (in the top left) or over (in the lower right) the central box.
- [x] Please make the _minimum_ font size in your diagram to be 10pt, and more ideally 11pt (space allowing), using Helvetica font (or similar).
- [x] Please add the labels “Stage I: Data Input”, “Stage II: Data Processing” and “Stage III: Data Analysis” to indicate the appropriate sections of your Diagram, following the examples in [CaseStudies123.png](https://github.com/BIDS/repro-case-studies/blob/submissions/CaseStudies123.png) as a guideline.  These labels should be in Helvetica font, bold, 22pt.
- [x] Please change any color/grayscale in your diagram to black & white.  You might consider using additional shaped boxes, or various outline styles for boxes (and adding a legend) to denote different types of information or tools.
- [x] Please keep your Diagram horizontally (landscape) oriented, and occupying no more than 1/2 of a US letter page.
- [x] Please provide both the PDF and editable source file for your Diagram.

Here is a traditional examples of a case study [Narrative](https://github.com/BIDS/repro-case-studies/blob/submissions/case-studies/dturek.md) and [Diagram](https://github.com/BIDS/repro-case-studies/blob/submissions/case-studies/dturek.pdf), which represent both the style and degree of "concreteness" that we're looking for.

**Comment by dani-lbnl** [comment]
Dear Daniel,

I believe I have addressed this issue. Please let me know if you find any problems. I've submitted a pull request.
thanks, Dani

**Comment by dani-lbnl** [comment]
Hi Daniel,
It seems that you want me to change the diagram, however the idea of stage I, II, and III does not quite apply to the SIPOC diagram. Should I just completely change that diagram and use your theme instead? thx, Dani

**Comment by fatmai** [comment]
@dani-lbnl , do you think the following labels could be suitable in your diagram 
Stage I: (Sources & Input) 
Stage II: Process
? thx, f

**Comment by danielturek** [comment]
@dani-lbnl, can you please create a new PR into the **submissions** branch of the BIDS/repro-case-studies repository?

Also the addition of a one-line title to your Narrative, as @fatmai mentioned.

And I hope you don't have to re-work the entire diagram.  But we are aiming for a consistent format using labels such as "Stage I: Data Input", and "Stage II: Data Processing".  Would these be suitable to you?

**Comment by dani-lbnl** [comment]
@fatmai @danielturek -- I am changing the diagram the best I can to accommodate requests, but because I use Lucidchart, most of the changes are impossible, for example
1) Please remove the little semi-circles in the lines, where they overlap. Not necessary, it’s understandable where the lines are going. >> it's inherent to the diagramming tool
2) Please make it so the lines from the top-left box (Sources) don’t cross. >>tried the best I could, but the boxes position force the connection in a certain way.
3) Please adjust the lines surrounding the central box (Software Tools) so they don’t go behind (in the top left) or over (in the lower right) the central box. >>same problem
4) Please make the minimum font size in your diagram to be 10pt, and more ideally 11pt (space allowing), using Helvetica font (or similar) >>> helvetica, arial and other common fonts are proprietary and not allowed within Lucidchart... I'm attaching the msg --- i could not believe myself...

**Comment by dani-lbnl** [comment]
![screen shot 2016-04-13 at 12 46 48 pm](https://cloud.githubusercontent.com/assets/8379306/14507311/df38b6c2-0176-11e6-8cd2-a8ea7f904f1b.png)

**Comment by danielturek** [comment]
@dani-lbnl Thanks for all the details.  In that case, can you please open a new PR into the submissions branch, with your updated Narrative (including title) and Diagram?

**Comment by dani-lbnl** [comment]
@danielturek @fatmai I've made many changes to the diagram trying to address the your comments, thanks. new PR on the way

**Comment by fatmai** [comment]
Thanks, @dani-lbnl for the revised diagram and title. There are few minor revisions left before we can merge your PR.
- [x] Could you please remove all the yellow and gray colors from your diagram? We would like to have all the diagrams consistent and use only black and white.
- [x] I read the label Stage I: Sources & Inputs, Stage II: Inputs & Process, Stage III: Process & Output & Consumer. I am not sure if it is intended that "Inputs" is in both "Stage I" and "Stage II" (similarly Process). Does it make sense to separate these stages in the following way?
  - Stage I: Source and Inputs 
  - Stage II and III: Process
- [x] Please provide along the PDF of your diagram an editable file such as SVG.
- [ ] Please submit your new PR to the submissions branch and not the master branch.
  
  I will close PR #100 and wait for a new PR in the submission branch. Thanks again!

**Comment by dani-lbnl** [comment]
Thanks for the comments @fatmai . I believe I've PR to the right branch now, and provided the fixes to the diagram accordingly.

**Comment by danielturek** [comment]
@dani-lbnl Thanks for working with us.

I just took a look, and it seems the diagram in PR #105 still has color and grayscale: a yellow tint to all the boxes, and grey background in the boxes along the top.

Are you able to adjust it one final time, to remove both the color and the grayscale?  We're aiming for strictly black & white diagrams.

Thanks!

**Comment by dani-lbnl** [comment]
@danielturek  - thanks for your reviews, I've removed color and grayscale. hope now it works. thx.

**Comment by danielturek** [comment]
Merged #109 with revisions to diagram.  Thank you @dani-lbnl, your case study revisions are complete.

**Comment by danielturek** [comment]
@dani-lbnl One very minor additional request for your case study digram.  I'm sorry, but I'm having trouble modifying your SVG diagram file.

Can you please change the text that reads "GITHUB" to instead read "GitHub"?

If you can submit the changed SVG and PDF files to the _submissions_ branch, that would be great.  Thanks!

**Comment by jkitzes** [comment]
After editor discussion, decided to leave this as is for now. Will return if needed when UC Press gives us more guidance on figure formats.
